test/run-envoy-unit-tests.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+set -e
+
+vagrant ssh runtime -c "cd /src; make tests-envoy"
```

Comment:
@ianvernon  I'm not sure that this works, with the new image the /src/ no longer exists

Reply:
Good catch, I'll fix that.